stellar-multitenant/pkg/cli/utils/custom_set_value.go

```diff
@@ -62,25 +61,6 @@ func SetConfigOptionURLString(co *config.ConfigOption) error {
 	return nil
 }
 
-func SetConfigOptionOptionalBoolean(co *config.ConfigOption) error {
-	b := viper.GetString(co.Name)
-	if b == "" {
-		return nil
-	}
-
-	value, err := strconv.ParseBool(b)
-	if err != nil {
-		return fmt.Errorf("parsing %q as a boolean value: %w", b, err)
-	}
-
-	key, ok := co.ConfigKey.(**bool)
-	if !ok {
-		return fmt.Errorf("the expected type for this config key is a boolean, but got a %T instead", co.ConfigKey)
-	}
-	*key = &value
-	return nil
-}
-
```

Comment:
FYI, we shouldn't need this method in the first place, since you can just make sure to use type.Bool like in:https://github.com/stellar/stellar-disbursement-platform-backend/blob/ac3c909a2b148a709970f9132fad77aa0c4b50cf/cmd/serve.go#L255-L262

Reply:
But that's an `Optional Boolean` config option. The problem is that `types.Bool` type requires a `default` value to be set. Since we **couldn't** have a default value because it would affect the tenant configuration, I created this `custom set value` function.

For instance, if the default value is `true`, and the user sets the value of the tenant config to `false`. In the next run if the user doesn't add this config again the CLI will set the `true` value for that config because it's the default value.